### ISSUE TITLE
Pass word_level_timestamps option into whisper API call

### ISSUE THREAD

**Issue by ffrenchm** [openai/openai-node#172]
### Describe the feature or improvement you're requesting

When I use the Whisper model directly via the openai/whisper python package (https://github.com/openai/whisper) I am able to pass the `word_timestamps` into the transcription call. See the below snippet:

```
whisper_model = whisper.load_model("large-v2").to(device)

transcript_res = whisper_model.transcribe(
  audio=f"{filename}.mp3",
  word_timestamps=True,
  language="en"
)
```

However, when I make an API call, I'm not able to do this. I find this a very useful feature, as by default, the timestamps in the response are rounded to the nearest second, whereas when passing in `word_timestamps=True` they're much more accurate.

Could the `word_timestamps` parameter be added to the API?

**Comment by oli5679** [comment]
I ran into the same issue a few days ago, and. I might be misreading the do me know if there is a suggested workaround. It would be even more helpful if there was some diarisation functionality directly built into the API.

**Comment by prafful1234** [comment]
umm, [this](https://github.com/openai/openai-node/blob/dc821be3018c832650e21285bade265099f99efb/api.ts#L2581) might be the reason as we only support `whisper-1` it dose not have `word_timestamps` option. 

Also author [reference](https://github.com/openai/whisper/discussions/3#discussioncomment-3703465).

**Comment by ffrenchm** [comment]
My understanding was that the "whisper-1" model was the "large-v2" model in https://github.com/openai/whisper, the whisper python package.

I'm currently using the large-v2 model from the openai-whisper python package. By setting the word_timestamps param to True, I get timestamps to the nearest millisecond, and word level timestamps. This is essential for my use case.

```
model = whisper.load_model(
    name="./models/whisper-large-v2.pt",
    in_memory=True,
)

transcription_response = model.transcribe(
    audio=local_audio_path,
    word_timestamps=True,
    verbose=False,
    language="en"
)
```

Ideally this functionality would be available via the API, however it isn't. Therefore I can't use the API and have had to host the model myself. Hopefully this feature can be added as I'm sure it would be helpful to other users.

**Comment by Francoyy** [comment]
Same here, I would need the following parameters to be handled:
`--word_timestamps True --max_line_width 56 --max_line_count 1`

Seems that I have no choice but to use Whisper locally for now. That's fine, but for a 10 minutes audio sequence, it takes 10 seconds remotely, VS 30 minutes locally.

**Comment by rattrayalex** [comment]
Thanks for the feature request; we're exploring adding this to the API.

In the meantime, as a workaround for use-cases which don't require word-level fidelity, you can get phrase-level timestamps with `response_format: "verbose_json"`, which returns results like this:

```js
{
  task: 'transcribe',
  language: 'english',
  duration: 10.04,
  text: 'It was anxious to find him no in that expectation of a man who were giving his father enjoyment. And he was avoided in sight in the minister, to which, indeed,',
  segments: [
    {
      id: 0,
      seek: 0,
      start: 0,
      end: 6.38,
      text: ' It was anxious to find him no in that expectation of a man who were giving his father enjoyment.',
      tokens: [
        50364,   467,   390, 15166,
          281,   915,   796,   572,
          294,   300, 14334,   295,
          257,   587,   567,   645,
         2902,   702,  3086, 32013,
           13, 50683
      ],
      temperature: 0,
      avg_logprob: -0.39801921844482424,
      compression_ratio: 1.325,
      no_speech_prob: 0.08255614340305328
    },
    {
      id: 1,
      seek: 0,
      start: 6.38,
      end: 10.06,
      text: ' And he was avoided in sight in the minister, to which, indeed,',
      tokens: [
        50683,   400,  415, 390,
        24890,   294, 7860, 294,
          264, 10563,   11, 281,
          597,    11, 6451,  11,
        50867
      ],
      temperature: 0,
      avg_logprob: -0.39801921844482424,
      compression_ratio: 1.325,
      no_speech_prob: 0.08255614340305328
    }
  ]
}
```

**Comment by raihanNoman** [comment]
I tested the phrase level.
It works too well. Like, I am displaying transcription text a whole 2-3 seconds before or after it's heard on the recording. 

IMO, it works alright about 30% of the time. Especially if you have silences between sentences, it considers them part of the phrase and results in an atrocious user experience. Trying to use a lamda function with whisper-timestamped. 

Been having some trouble. When I succeed, I will post it here...
